tiledb/sm/rest/rest_profile.h

```diff
@@ -185,6 +233,23 @@ class RestProfile {
       std::make_pair(
           "rest.server_address", RestProfile::DEFAULT_SERVER_ADDRESS),
       std::make_pair("rest.username", RestProfile::DEFAULT_USERNAME)};
+
+  /**
+   * Flag representing whether or not this RestProfile has been `save`d.
+   * When `true`, this profile has been immutably written to disk, and its
+   * parameters can be fetched from upstream (the parent Config).
+   */
+  bool saved_{false};
+
+  /**
+   * Flag which tracks the `Config::verify_ssl` parameter inherited from
+   * `cloud.json`.
+   *
+   * @note This is a temporary workaround, to be removed once the cloud API
```

Comment:
I remember we discussed this, I don't fully remember the discussion - this comment isn't quite enough to crystallize it for me.

And I'm fine with that since eventual removal of this is implied here - what's the sequence there?

1) merge the API
2) merge this
3) ???
4) remove this parameter

Reply:
Ah, right, the verify SSL parameter is historically stored in `cloud.json` but is not meant to be part of the profile - correct?